src-tauri/tauri.conf.json

```diff
@@ -1,7 +1,6 @@
 {
   "$schema": "../node_modules/@tauri-apps/cli/schema.json",
   "build": {
-    "beforeBuildCommand": "yarn build:both",
```

Comment:
@Irev-Dev Had to remove this and run it manually from the CI so I had control over the [mode](https://vitejs.dev/guide/env-and-mode).

Let me know if you think we should do this another way.

Another PR should probably clean up the pile of scripts we have in `package.json`, don't believe all of them have a purpose / aren't redundant.

Reply:
Dope. Future issue at https://github.com/KittyCAD/modeling-app/issues/1188

Thanks for the quick review!